clang/runtime/dpct-rt/include/dpct/graph.hpp

```diff
@@ -0,0 +1,26 @@
+//==---- graph.hpp ----------------------------*- C++ -*----------------==//
+//
+// Copyright (C) Intel Corporation
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <sycl/ext/oneapi/experimental/graph.hpp>
+#include <sycl/sycl.hpp>
+
+namespace dpct {
+namespace experimental {
+
+typedef std::shared_ptr<sycl::ext::oneapi::experimental::command_graph<
+    sycl::ext::oneapi::experimental::graph_state::modifiable>>
+    command_graph_t;
+
+typedef std::shared_ptr<sycl::ext::oneapi::experimental::command_graph<
+    sycl::ext::oneapi::experimental::graph_state::executable>>
+    command_graph_exec_t;
```

Comment:
Not sure whether should the variable name `command_graph_t` and `command_graph_exec_t` need changed to xxx_ptr.